### PR TITLE
update loading CSS

### PR DESCRIPTION
**Pull request by mitchell-frost**
_I have added a description about the loading of CSS. The order was not mentioned before. it is for the issue #3831 

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/content/guides/asset-management.md [hunk 1]
```diff
@@ -86,6 +86,9 @@ __webpack.config.js__
 +   },
   };
 ```
+Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain. 
+
+The above order or loaders should be maintained with 'style-loader' coming first and then, 'css-loader'. If this convention is not followed, webpack is likely to throw errors.
```
Comment (EugeneHlushko) [inline src/content/guides/asset-management.md:91]:
```suggestion
The above order of loaders should be maintained: 'style-loader' comes first and followed by 'css-loader'. If this convention is not followed, webpack is likely to throw errors.
```

File: src/content/guides/asset-management.md [hunk 2]
```diff
@@ -86,6 +86,9 @@ __webpack.config.js__
 +   },
   };
 ```
+Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain.
```
Comment (EugeneHlushko) [inline src/content/guides/asset-management.md:89]:
```suggestion

Module loaders can be chained. Each loader in the chain applies transformations to the processed resource. A chain is executed in reverse order. The first loader passes its result (resource with applied transformations) to the next one, and so forth. Finally, webpack expects JavaScript to be returned by the last loader in the chain. 
```

File: src/content/guides/asset-management.md (hunk 1 above)
Comment (mitchell-frost) [reply src/content/guides/asset-management.md:91]: I made the changes required. I actually could not reference this branch and hence, I created another pull request. I hope that's okay.

File: src/content/guides/asset-management.md (hunk 2 above)
Comment (mitchell-frost) [reply src/content/guides/asset-management.md:89]: I made the changes.

**Comment by EugeneHlushko** [comment]
Closed via #3913

**Pull request closed without merging**